pom.xml

```diff
@@ -447,7 +447,26 @@
 
       <dependency>
         <groupId>org.apache.beam</groupId>
-        <artifactId>beam-sdks-java-io-elasticsearch</artifactId>
+        <artifactId>beam-sdks-java-io-elasticsearch-2</artifactId>
```

Comment:
not ideal indeed (see my comment bellow), but at least user pipeline code is the same, only the user pom differs.